README.md

```diff
@@ -369,17 +354,16 @@ The following apply when activating an installation (i.e. `. ${KERL_DEFAULT_INST
 
 When set, automatically prefix the shell prompt with a section containing the erlang version (see [`$KERL_PROMPT_FORMAT`](#kerl_prompt_format) ).
 
-
 ### KERL_PROMPT_FORMAT
 
 Default: `(%BUILDNAME%)`
 Available variables:
+
  - `%BUILDNAME%`: Name of the kerl build (e.g. `my_test_build_18.0`)
  - `%RELEASE%`: Name of the erlang release (e.g. `19.2` or `R16B02`)
 
 The format of the prompt section to add.
 
-
 ### KERL_DEFAULT_INSTALL_DIR
```

Comment:
See this should probably be a "Install configuration" item, no?

Reply:
I don't know. I just fixed the `markdownlint` -specific issues. I tried to not broaden the scope of the pull request by not reviewing the README. Shall I change something here? Or can this be postponed to a future issue?

Reply:
Yeah it's fine to sort through it later